src/openms/include/OpenMS/COMPARISON/SPECTRA/SpectrumAlignment.h

```diff
@@ -51,9 +51,10 @@ namespace OpenMS
   /**
       @brief Aligns the peaks of two spectra
 
-        Using a banded (width via 'tolerance' parameter) alignment.
+        Using a banded (width via 'tolerance' parameter) alignment if absolute tolerances are given.
         Scoring function is the m/z distance between peaks. Intensity does not play a role!
 
+        If relative tolerance (ppm) is given simple matching of peaks is performed.
```

Comment:
maybe extend the docs here:
e.g., matching the closest peak; with s1 serving as reference.
Implications: double assignment of peaks in s2 possible; missing assignments in either s1 or s2 possible